engine/gamesys/src/gamesys/scripts/script_tilemap.cpp

```diff
@@ -485,7 +501,21 @@ namespace dmGameSystem
     void ScriptTileMapRegister(const ScriptLibContext& context)
     {
         lua_State* L = context.m_LuaState;
+        DM_LUA_STACK_CHECK(L, 0);
         luaL_register(L, "tilemap", TILEMAP_FUNCTIONS);
+
+        #define SETCONSTANT(name, val) \
+            lua_pushnumber(L, (lua_Number) val); \
+            lua_setfield(L, -2, #name);\
+
+        SETCONSTANT(H_FLIP, 1);
+        SETCONSTANT(V_FLIP, 2);
+        SETCONSTANT(ROTATE_90, 4);
+        SETCONSTANT(ROTATE_180, -3);
```

Comment:
THis is a bit unconventional?

Why not:
ROTATE_90 4
ROTATE_180 8
ROTATE_270 16

Reply:
Another option is to set R180 = H_FLIP + V_FLIP, which is much more readable